### DISCUSSION TITLE
Nodejs download extraction Chrome book OS

### DISCUSSION THREAD

**Discussion by codegenerations** [community/community#31324] [Other Feature Feedback, Questions, & Ideas]
Dear all,

Which version after clicking ´Recommended for most users´: https://nodejs.org/en/ I need to take to download NodeJS for my chromebook OS Linux?
Because after downloading i get files. but the download setup wizard do not show up on my screen?

Kind regards,

Me.

**Comment by jfullstackdev** [comment]
I'm not sure of this one but these two articles from the same author might help

https://medium.com/@jacoboakley/web-development-with-a-chromebook-installing-nodejs-4e358b82a31b

https://medium.com/@jacoboakley/web-development-with-a-chromebook-terminal-setup-dc0d4668d411

**Comment by codegenerations** [comment]
heyy Sir,

I read it.
I want to install visual studio code on this Microsoft site:
https://code.visualstudio.com/docs/setup/linux/

I think I have Debian/Ubuntu. But when i enter this code: in my terminal.
It goes wrong with this line: ¨ -qo- ¨ the terminal say this: ¨
configuration item specification must have an =<val> ¨

What to do about this? can you help me?

Kind regards,

sudo apt-get install wget gpg

wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg
--dearmor > packages.microsoft.gpgsudo install -D -o root -g root -m
644 packages.microsoft.gpg
/etc/apt/keyrings/packages.microsoft.gpgsudo sh -c 'echo "deb
[arch=amd64,arm64,armhf
signed-by=/etc/apt/keyrings/packages.microsoft.gpg]
https://packages.microsoft.com/repos/code stable main" >
/etc/apt/sources.list.d/vscode.list'rm -f packages.microsoft.gpg






Op wo 31 aug. 2022 om 16:33 schreef jdevstatic ***@***.***>:

**Comment by codegenerations** [comment]
Dear Sir,

Next question is.. Would you like to share your number?

Have a nice weekend

Kind regards,

Op wo 31 aug. 2022 om 16:33 schreef jdevstatic ***@***.***>:
